install_config/storage_examples/containerized_heketi_with_dedicated_gluster.adoc

```diff
@@ -0,0 +1,386 @@
+====
++
+[source, json]
+----
+{
+  "clusters": [
+    {
+      "nodes": [
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster23.rhs"
+              ],
+              "storage": [
+                "192.168.1.200"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        },
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster24.rhs"
+              ],
+              "storage": [
+                "192.168.1.201"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        },
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster25.rhs"
+              ],
+              "storage": [
+                "192.168.1.202"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        }
+      ]
+    }
+  ]
+}
+----
+
+. Using `heketi-cli`, run the following command to load the topology of your
+environment.
++
+[source, bash]
+----
+$ heketi-cli topology load --json=toplogy-sample.json
+
+    	Found node gluster23.rhs on cluster bdf9d8ca3fa269ff89854faf58f34b9a
+   		Adding device /dev/sde ... OK
+   	 	Adding device /dev/sdf ... OK
+    	Creating node gluster24.rhs ... ID: 8e677d8bebe13a3f6846e78a67f07f30
+   	 	Adding device /dev/sde ... OK
+   	 	Adding device /dev/sdf ... OK
+...
+----
+
+. Create a Gluster volume to verify Heketi:
++
+[source, bash]
+----
+$ heketi-cli volume create --size=50
+----
+
+. View the volume information from one of the the Gluster nodes:
++
+[source, bash]
+----
+$ gluster volume info
+
+	Volume Name: vol_335d247ac57ecdf40ac616514cc6257f <1>
+	Type: Distributed-Replicate
+	Volume ID: 75be7940-9b09-4e7f-bfb0-a7eb24b411e3
+	Status: Started
+...
+----
+<1> Volume created by `heketi-cli`.
+
+[[dedicated-glusterfs-dynamic-provision-volume]]
+== Dynamically Provision a Volume
+[NOTE]
+====
+If you installed {product-title} by using the link:https://github.com/openshift/openshift-ansible/tree/master/inventory/byo[BYO (Bring your own) OpenShift Ansible inventory configuration files] for either link:https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.byo.glusterfs.native.example[native] or link:https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.byo.glusterfs.external.example[external] GlusterFS instance, the GlusterFS storageClass automatically get created during the installation. For such cases you can skip the following storageclass creation steps and directly proceed with creating persistent volume claim instruction.
+====
```

Comment:
Should the last storageclass has a capital C? "...skip the following storageClass creation steps..."

Reply:
I think we should use `StorageClass` when we talk about specific object and use storage class in general.